client/components/domains/register-domain-step/index.jsx

```diff
@@ -170,6 +170,8 @@ class RegisterDomainStep extends React.Component {
 		onAddDomain: PropTypes.func,
 		onAddTransfer: PropTypes.func,
 		designType: PropTypes.string,
+		recordFilterSubmit: PropTypes.func.isRequired,
```

Comment:
These two props are throwing warnings in the console:
```
Warning: Failed prop type: The prop `recordFilterSubmit` is marked as required in `RegisterDomainStep`, but its value is `undefined`.
```

To reproduce - go to Domain Search in Domain Management.

Additionally, looks like they are not used anywhere anymore?

Reply:
Ah, I need to update those two props to be `recordFiltersSubmit` and `recordFiltersReset` instead.